scargo/commands/test.py

```diff
@@ -66,7 +67,16 @@ def run_ut(config: Config, verbose: bool, cwd: Path) -> None:
         subprocess.run(cmd, cwd=cwd, check=False)
 
         # Run code coverage.
-        cmd = ["gcovr", "-r", "ut", ".", "--html", "ut-coverage.html"]
+        cmd = [
+            "gcovr",
+            "-r",
+            "ut",
+            ".",
+            "-f",
+            cwd.parents[1].joinpath("src"),
```

Comment:
this seems to be equivalent to `config.project_root / "src"`, which in turn is incorrect, because in esp projects there's `main` directory instead of `src`. I think it should be `config.project_root / config.project.target.source_dir` instead.